### PR TITLE
Save the credentials in the current session

### PR DESCRIPTION
**Pull request by bijayprasadtiwari**
Save the credentials in the current session, do not save the Credentials in file.

### CODE REVIEW

File: src/Cohesity.Powershell/Scripts/Utility/Invoke-RestApi.ps1 [hunk 1]
```diff
@@ -101,13 +101,15 @@ function Invoke-RestApi
...
                 $cohesitySession = Get-Content -Path $HOME/.cohesity | ConvertFrom-Json
-                if($null -ne $cohesitySession.Credentials) {
+                if($null -ne $credentialsJson) {
                     $cohesityUrl = $cohesitySession.ClusterUri + "/irisservices/api/v1/public/accessTokens"
+					$credentialsObject = $credentialsJson | ConvertFrom-Json
                     $payload = @{
-                        Domain                 = $cohesitySession.Credentials.Domain
-                        Username                   = $cohesitySession.Credentials.Username
-                        Password                   = $cohesitySession.Credentials.Password
+                        Domain                 = $credentialsObject.Domain
```
Comment (ashish-cohesity) [inline src/Cohesity.Powershell/Scripts/Utility/Invoke-RestApi.ps1:110]: Good stuff, what about the C# based cmdlets ?

Comment (bijayprasadtiwari) [reply src/Cohesity.Powershell/Scripts/Utility/Invoke-RestApi.ps1:110]: The C# cmdlete has been covered on the RestApiClient.cs

Comment (ashish-cohesity) [reply src/Cohesity.Powershell/Scripts/Utility/Invoke-RestApi.ps1:110]: LGTM

**Comment by ashish-cohesity** [comment]
LGTM

**Pull request merged**